spec/luassert_helper.lua

```diff
@@ -1,18 +1,28 @@
 local busted = require('busted')
-local assert = require("luassert")
+local assert = require('luassert.assert')
 local util = require ('luassert.util')
 local say = require('say')
 
 do -- set up reverting stubs
-  local snapshot
+  local state = require("luassert.state")
```

Comment:
Before this change we were calling `require('luassert'):snapshot()`. Is this equivalent to `require('luassert.state').snapshot()` ? What are the differences?

Reply:
Yep. `state.snapshot()` creates one and stores it on a stack, so you don't need to have a reference to it. You can just push/pop from the stack. 

With `:snapshot()` and `:revert()` it was increasingly hard to keep the state somewhere (like on a stack) and it uses luassert.state underneath anyway.